### PR TITLE
Updated read_file for unstructured-io

### PR DESCRIPTION
**Pull request by AdarshJha619**
### Description

Updated read_file using unstructured-io

### Solution and Design

Add unstructured library in your requirements.txt file 
Install the corresponding dependency: libmagic-dev, poppler-utils, tesseract-ocr, libreoffice, pandoc

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update

### Checklist
- [x] My pull request is atomic and focuses on a single change.
- [ ] I have read the contributing guide and my code conforms to the guidelines.
- [ ] I have documented my changes clearly and comprehensively.
- [ ] I have added the required tests.

### CODE REVIEW

**Review by Fluder-Paradyne — APPROVED** [review]

File: tools.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
   "tools": {
   }
-}
```
Comment (nborthy) [inline tools.json:4]: remove tools.json changes

Comment (AdarshJha619) [reply tools.json:4]: Done

Comment (nborthy) [reply tools.json:4]: remove this file from your changes, exclude it in this PR. Dont delete the content of the file

Comment (AdarshJha619) [reply tools.json:4]: check now

**Review by nborthy — APPROVED** [review]

**Pull request merged**